src/orchestrator/types.ts

```diff
@@ -282,6 +285,50 @@ interface TokenConfig {
   balanceSlot: (address: Address) => Hex
 }
 
+export enum MainnetNetwork {
```

Comment:
Instead of duplicating this we can create ids list by going through mainnets testnets already defined from viem chain objects